modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -16,20 +16,20 @@ To download the Anypoint Service Mesh client, type the following command at the
 `curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`
 
 
-== Configure Your Environment Variable
+// == Configure Your Environment Variable
```

Comment:
Configure or update? 
Choose the more precise of the two and use it consistently throughout this section...which is commented out?

Reply:
Done! Thanks!